package.json

```diff
@@ -182,7 +181,7 @@
     "redux-form": "^8.0.3"
   },
   "peerDependencies": {
-    "@folio/stripes": "^6.2.0",
+    "@folio/stripes": "^7.0.0",
```

Comment:
Either you need to update the dev-deps react and react-dom to ^17 as well or handle that in a separate PR. A separate PR is probably cleaner but 🤷 . I've been changing `"react": "*"` to `^17.0.2` as well; claiming compatibility with all unknown future releases of react seems, hmmmm, overly optimistic 😆 .

Reply:
Good point!